src/contents/docs/themes/content-types/index.md

```diff
@@ -0,0 +1,56 @@
+---
+type: guide
+title: Content types
+description: Content types
+category: themes
+order: 6
+---
+
+# Content types
+---
+
+Custom content types are defined in the `types` folder as simple YAML files.
+
+A content type can have a specific location where Markdown files under that path are recognized as that type, or the type can be specified in the front matter.
+
+A content type may also have a custom template, allowing the system to use the appropriate [Mustache](/docs/themes/mustache-templates/) file when rendering specific content, such as a blog post.
+
+Here's an example:
+
+```yaml
+id: post
+rss: true
+template: blog.post.default
+
+pagination:
+    bundle: blog/posts/pagination
+    limit: 9
+    sort: publication
+    order: desc
+
+properties:
+    featured:
+        type: bool
+        required: true
```

Comment:
This has been removed as properties are now required by default. However, this isn’t an issue.

Reply:
```suggestion
```